db/seeds/user.rb

```diff
@@ -1,7 +1,13 @@
-gds_organisation_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+if User.where(name: "Test user").present?
+  puts "Skipping because user already exists"
+else
+  gds_organisation_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+  user_id = "0d825034-6ea5-49e9-98a0-167ea2a6c487"
```

Comment:
Does it matter which uid this is? i.e. the gds one is the same one that matches the [actual GDS one](https://www.gov.uk/api/organisations/government-digital-service) - if not might be more revealing of intent to use `SecureRandom.uuid`

Reply:
Sounds good 👍